prisma/schema.prisma

```diff
@@ -48,6 +48,7 @@ model User {
   name          String
   email         String        @unique
   emailVerified DateTime?
+  role          Int        @default(0)
```

Comment:
What is the role 0?

Reply:
a normal user would be a 0, an admin would be a 1, maybe we could add a reader-admin role, but could add that easy with the enum.ts